src/PharoThings-Devices-HD44780.package/PotHD44780Controller.class/instance/bitOrArray..st

```diff
@@ -1,9 +1,9 @@
 initialization
 bitOrArray: anArray
-	
 	| hex hexOr |
-	1 to: anArray size do: [ :i |  
-		hex := (instructionDic at: (anArray at: i )) .
-		hexOr ifNil: [ hexOr := hex] ifNotNil: [ hexOr := hexOr bitOr: hex ].
-		 ].   
-	^hexOr. 
\ No newline at end of file
+	1 to: anArray size do: [ :i | 
+		hex := anArray at: i.
+		hexOr
+			ifNil: [ hexOr := hex ]
+			ifNotNil: [ hexOr := hexOr bitOr: hex ] ].
```

Comment:
I think simple version will be interesting for you:
```Smalltalk
   ^anArray inject: 0 into:  [:each : hex | hex bitOr: each]
```

Reply:
Also method name is not really clear. It converts array of hex values to number, right? Then #hexArrayToNumber: would be much better